bin/process_output.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python
+
+from pathlib import Path
+from mimetypes import guess_type
+from functools import partial
+import gzip
+import sys
+import argparse
+import pandas as pd
+
+
+def get_open(f):
+    if "gzip" == guess_type(str(f))[1]:
+        return partial(gzip.open)
+    else:
+        return open
+
+def main(argv=None):
+
+    parser = argparse.ArgumentParser(
+        description="Parses a profile_dists distances to create query-reference-format output for the FastMatch pipeline.",
+        epilog="Example: python process_output.py --input distances.tsv --output results.tsv --threshold 10",
+    )
+
+    parser.add_argument(
+        "--input",
+        action="store",
+        dest="input",
+        type=str,
+        help="profile_dists-generated distance matrix",
+        default=None,
+        required=True,
+    )
+
+    parser.add_argument(
+        "--threshold",
+        action="store",
+        dest="threshold",
+        type=int,
```

Comment:
Threshold should be a float to account for percentage thresholds (e.g., percentage of allelic differences across scheme).

Reply:
Update in 4c18be30fbc6ee6c2e50fb7465cbd86df9062c3d